CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 2.8.11)
-project(exit2destination LANGUAGES C CXX)
+project(osrm-tag-rewriter LANGUAGES C CXX)
```

Comment:
all of these project changes look unrelated to the main goal of this PR. Should we maybe split them off into a separate commit?